spring-cloud-kubernetes-client-discovery/src/main/java/org/springframework/cloud/kubernetes/client/discovery/KubernetesClientPodLabelsAndAnnotationsSupplier.java

```diff
@@ -53,13 +53,6 @@ static KubernetesClientPodLabelsAndAnnotationsSupplier nonExternalName(CoreV1Api
 		return new KubernetesClientPodLabelsAndAnnotationsSupplier(coreV1Api, namespace);
 	}
 
-	/**
-	 * to be used when .spec.type of the Service is == 'ExternalName'.
```

Comment:
this PR was about only removing this method, but then `mvn clean install` locally re-formatted the other files, nothing changed there.